src/web/screens/sharingPermissions.tsx

```diff
@@ -1,9 +1,13 @@
+/* eslint-disable camelcase */
```

Comment:
I think we can remove this? There's non-camelcase being used but they are defined in a separate file (i.e. `adminServiceHelpers.ts`)

Reply:
Good catch - I had moved them into this file but ended up getting rid of them again.